lib/log.js

```diff
@@ -3,11 +3,11 @@ import rfs from 'rotating-file-stream'
 import { basename, dirname } from 'node:path'
 import { paths } from './paths.js'
 
-export const formatLog = text =>
+export const formatLog = (text, date = new Date()) =>
   text
     .trimEnd()
     .split(/\n/g)
-    .map(line => `[${new Date().toLocaleTimeString()}] ${line}`)
+    .map(line => `[${date.toLocaleString()}] ${line}`)
```

Comment:
changed the date format so date strings can be parsed back to date objects, which wasn't possible before. This isn't a breaking change

Reply:
Do we need to update any example output in README to accommodate this change?